tpm2/encoding_test.go

```diff
@@ -385,6 +385,14 @@ func TestECCParamsEncodeDecode(t *testing.T) {
 		t.Fatalf("decodeECCParams: %v", err)
 	}
 
+	if params.Point.X().Cmp(pk.PublicKey.X) != 0 || params.Point.Y().Cmp(pk.PublicKey.Y) != 0 {
+		t.Fatalf("Point deserialized: (%d, %d)\nGenerated: (%d, %d)", params.Point.X(), params.Point.Y(), pk.PublicKey.X, pk.PublicKey.Y)
```

Comment:
For these `FatalF` messages, just use the shorter `"got: %+v\nwant: %+v"` format string, like we do below.

Reply:
Done